docker-compose.yml

```diff
@@ -98,3 +112,27 @@ services:
       - /var/log/multi-cloud:/var/log/multi-cloud
     ports:
       - 27017:27017
+
+  tidb:
+    image: pingcap/tidb:v2.1.16
+    container_name: tidb
+    ports:
+      - "4000:4000"
+      - "10080:10080"
+    volumes:
+      - ./s3/tidbconf/tidb.toml:/tidb.toml:ro
+    command:
+      - --store=mocktikv
+      - --config=/tidb.toml
+      - -L=info
+
+  mysql:
+    image: mysql:5.7
+    container_name: mysql
+    volumes:
+      - ./s3/pkg/conf/tidb.sql:/yig.sql
+    command: mysql -h tidb  -P 4000 -u root -e "create database yig character set utf8;use yig;source /yig.sql"
+    environment:
+      - MYSQL_ALLOW_EMPTY_PASSWORD=yes

```

Comment:
this is a security hole

Reply:
thanks @rhsakarpos for your review, and the changes here in docker-compose.yml is only for test usage. For the deployment of multi-cloud, there will be a deployment team who will work for it.

Reply:
please raise issue and assign to deployment owner, so we do not miss this, thanks

Reply:
@rhsakarpos  thanks,  I have add a issue to track it,  #683